### DISCUSSION TITLE
FirstInputFocus always on modal open

### DISCUSSION THREAD

**Discussion by mkts02** [wire-elements/modal#367] [Q&A]
can it be disabled?
thanks.

**Comment by richcarni** [comment]
It looks like this behaviour comes from Alpine.js x-trap and currently can't be configured: [discussed here](https://github.com/alpinejs/alpine/discussions/2806)

I'm going to have a go at a PR with something like a .noautofocus modifier for x-trap, and then we can hopefully add an option here

**Reply by richcarni** [reply]
Alpine.js PR: [https://github.com/alpinejs/alpine/pull/3977](https://github.com/alpinejs/alpine/pull/3977)
